src/libs/reportUtils.js

```diff
@@ -11,7 +12,21 @@ function getReportParticipantsTitle(logins) {
     return _.map(logins, login => Str.removeSMSDomain(login)).join(', ');
 }
 
+/**
+ * Check whether a report action is Attachment is not.
+ *
+ * @param {Object} reportAction action from a Report
+ * @returns {Boolean}
+ */
+function isReportActionAttachment(reportAction) {
+    if (_.has(reportAction, 'isAttachment')) {
+        return reportAction.isAttachment;
+    }
+    const lastMessage = _.last(lodashGet(reportAction, 'message', null));
+    return lodashGet(lastMessage, 'text', '') === '[Attachment]';
+}
```

Comment:
I think this would be more useful as a simple string method tbh, then we can use this same utility method in these spots:

https://github.com/Expensify/Expensify.cash/blob/276279823d774509eed8cd0df9bf486b84f1a965/src/libs/actions/Report.js#L412
https://github.com/Expensify/Expensify.cash/blob/276279823d774509eed8cd0df9bf486b84f1a965/src/libs/actions/Report.js#L150

Reply:
The goal is to make few references to `[Attachment]`

Reply:
Do you want me to still check for the `isAttachment` key in the `onPress` before fallback to this check?

Reply:
yeah I think what you have now works.